src/lib.rs

```diff
@@ -198,6 +198,6 @@ pub use ioctls::vm::{IoEventAddress, NoDatamatch, VmFd};
 ///
 /// ```
 /// #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-/// use kvm_ioctls::{KvmRunWrapper, Result};
+/// use kvm_ioctls::KvmRunWrapper;
```

Comment:
Can you also add a comment in the commit message to say that the Result is no longer exported and the reason behind that?

Reply:
done